daos/mysql/Database.php

```diff
@@ -243,6 +243,12 @@ public function __construct() {
                         'INSERT INTO ' . \F3::get('db_prefix') . 'version (version) VALUES (12)'
                     ]);
                 }
+                if (strnatcmp($version, '13') < 0) {
+                    \F3::get('db')->exec([
+                        'ALTER IGNORE TABLE ' . \F3::get('db_prefix') . 'items ADD UNIQUE INDEX (uid (150), source);',
```

Comment:
It looks like `ALTER IGNORE TABLE` syntax is [not supported](https://dev.mysql.com/doc/refman/5.6/en/alter-table.html#idm139885224435136) any more in MySQL 5.7. Manual deduplication will be needed.

Reply:
fix in 7040883bfbd248dae6c5d266912379c31cfcacd5